spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/MaskHead.scala

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.nn.abstractnn.AbstractModule
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.{T, Table}
+
+class MaskHead(
+  val inChannels: Int = 0,
+  val resolution: Int = 0,
+  val scales: Array[Float],
+  val samplingRratio: Float = 0.1f,
+  val layers: Array[Int],
+  val dilation: Int = 1,
+  val numClasses: Int = 81, // coco dataset class number
```

Comment:
COCO dataset class number by default ?

Reply:
yeah, I use parameter values of COCO to initial. So here 81 for COCO (80 foreground + 1 background)